### ISSUE TITLE
[v3] Using string literals in XmlBuilder gives empty result

### ISSUE THREAD

**Issue by dpakach** [pact-foundation/pact-js#677]
### Software versions

_Please provide at least OS and version of pact-js_

- **OS**: ubuntu 20.04
- **Consumer Pact library**:  Pact JS 10.0.0-beta.36
- **Node Version**: v12.21.0

### Issue Checklist

Please confirm the following:

- [x] I have upgraded to the latest
- [ ] I have the read the FAQs in the Readme
- [x] I have triple checked, that there are **no unhandled promises** in my code
- [x] I have set my log level to debug and attached a log file showing the complete request/response cycle
- [ ] For bonus points and virtual high fives, I have created a reproduceable git repository (see below) to illustrate the problem

### Description
Creating interaction with XML body using the XmlBuilder, we cannot use simple strings on the XML
it must always be added as `MaatchersV3.*()`

eg, this interaction
``` 
.willRespondWith({
      status: 200,
      headers: applicationXmlResponseHeaders,
      body: new XmlBuilder('1.0', '', 'ocs').build(ocs => {
        ocs.appendElement('meta', '', (meta) => {
          meta.appendElement('status', '', 'ok')
            .appendElement('statuscode', '', '100')
            .appendElement('message', '', 'OK')
        }).appendElement('data', '', (data) => {
          data.appendElement('id', '', MatchersV3.string(user))
          data.appendElement('display-name', '', MatchersV3.regex(`(${user}|${displayName})`, user))
          data.appendElement('email', '', MatchersV3.regex(`(${user}@example\\..*)?`, ''))
        })
      })
    })
```

### Expected behaviour
```
<?xml version='1.0'?>
  <ocs>
    <meta>
      <status>ok</status>
      <statuscode>100</statuscode>
      <message>OK</message>
    </meta>
    <data>
       <id>test123</id>
       <display-name>test123</display-name>
       <email>test123</email>
     </data>
</ocs>
```
### Actual behaviour
the above interaction will return the body
```
<?xml version='1.0'?>
  <ocs>
    <meta>
      <status></status>
      <statuscode></statuscode>
      <message></message>
    </meta>
    <data>
       <id>test123</id>
       <display-name>test123</display-name>
       <email></email>
     </data>
</ocs>
```
**note that status, status code and message properties are empty**

_fill in here_

### Steps to reproduce

_How can someone else reproduce this bug?_

_For extra bonus points and internet karma, provide a repository, gist or reproducible code snippet so that we can test the problem._

_We recommend forking the project and modifying a relevant example [in the examples folder](https://github.com/pact-foundation/pact-js/blob/master/examples)_

### Relevant log files

_Please ensure you set logging to `DEBUG` and attach any relevant log files here (or link to a gist)._

https://gist.github.com/dpakach/d85379735411a1ee15f1f1308dd14bbd

**Comment by individual-it** [comment]
this used to work with `Pact JS 10.0.0-beta.33`

**Comment by TimothyJones** [comment]
Apologies if I am missing something- the reported actual behaviour looks correct to me- would you expect the fields to be empty? Or, are the expected/actual accidentally the wrong way around?

**Comment by individual-it** [comment]
The first version on the post had the expectation and the actual result mixed up, should be fixed now

I think the issue is just a typo in here https://github.com/pact-foundation/pact-js/blob/c74c6cd05f5101fdf249216099766913eedfd51f/src/v3/xml/xmlElement.ts#L59
`content` vs `context` I will fix it

**Comment by individual-it** [comment]
fix in #678
